src/snowflake/snowpark/modin/plugin/_internal/ordered_dataframe.py

```diff
@@ -537,7 +537,7 @@ def _get_active_column_snowflake_quoted_identifiers(
         return column_quoted_identifiers
 
     def _extract_quoted_identifiers_from_column_or_name(
-        self, col: ColumnOrName
+        self, col: ColumnOrName, active_columns: list[str]
     ) -> list[str]:
         """
```

Comment:
Can you update the doc string by providing a description for both arguments?

Reply:
Yes, updated